rpcs3/main.cpp

```diff
@@ -451,12 +450,12 @@ int main(int argc, char** argv)
 	// but I haven't found an implicit way to check for style yet, so we naively check them both here for now.
 	const bool use_cli_style = find_arg(arg_style, argc, argv) || find_arg(arg_stylesheet, argc, argv);
 
-	QScopedPointer<QCoreApplication> app(createApplication(argc, argv));
+	static QScopedPointer<QCoreApplication> app(createApplication(argc, argv));
```

Comment:
I think normal application should not outlive main()

Reply:
Fixed.